src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
+    <h1>{{ userName }}</h1>
+    <div id="totalCount">
+      <p>{{ totalRepoCount }} repositories</p>
+      <p>|</p>
+      <p>{{ totalRepoStarCount }} stars</p>
+    </div>
+    <hr>
+    <p v-if="isEmptyRepoList">
+      레포지토리가 존재하지 않습니다.
+    </p>
+    <repo-list-item
+      v-else
+      v-for="(repoListItem,idx) in repoList"
+      :key="idx"
```

Comment:
```suggestion
      :key="repoListItem.id"
```
는 어떠신가요?! ㅎㅎ

https://kr.vuejs.org/v2/guide/list.html#Maintaining-State